sdk-setup/src/mb2

```diff
@@ -244,9 +243,31 @@ COMMANDS
         project directory. (The file name part is ignored - project file
         selection is done by qmake call inside the .spec file.)
 
+    cmake [<project-dir>] [[--] <cmake-extra-args>]
```

Comment:
cmake requires that the project dir is the last argument. I think we should keep the same requirement. So the format for cmake invokations should be "cmake <cmake-extra-args> <project-dir>".

Reply:
mb2 needs to restrict it to either first or last argument to be able to recognize it reliably between all the possible arguments (there may be options accepting paths as arguments). cmake actually does not insist on argument order in this case, and qtc invokes it exactly like that - project dir first, options later. That's why mb2 requires it to be the very first argument. The same applies to the qmake command.